### ISSUE TITLE
Published on Docker Hub?

### ISSUE THREAD

**Issue by rugk** [wmacevoy/latex-docker#2]
Is this published on Docker Hub? if so, under which name? (If so, maybe also add this to the Readme, please.)

If not, maybe consider publishing it there…

**Comment by wmacevoy** [comment]
I don't think it is a good candidate - the point is to automate the building custom dockers for latex projects, there isn't just one; it's a user-specific family.

**Comment by GregHilston** [comment]
@wmacevoy I know its been three years, but can you elaborate on why this isn't a good candidate for dockerhub a bit more?

**Comment by wmacevoy** [comment]
Well, I thought there should be lots of base versions; one for each specific latex use case.  In retrospect there is probably room for a "small" and "large" version as a container.  I also didn't want the security of the dervied system to depend on the security of releases on this system; you can "rebase" your container but you don't have to.

The base install is pretty basic.  The added tools (like symlinks to all the commands) is custom not-dockerisms.  It may be better to use that; but still not convinced.
